### ISSUE TITLE
How do I decrypt encrypted documents from reports

### ISSUE THREAD

**Issue by zhumo1998** [amzn/selling-partner-api-docs#1140]
I requested a report document through the report API, and the API returned both IV and key. How can I decrypt this Amazon document through AES？

**Comment by gaurav-webs** [comment]
@zhumo1998 

I have used this code to decrypt the document and its work for me,

`openssl_decrypt(file_get_contents($feedDownloadUrl), 'aes-256-cbc', $key, OPENSSL_RAW_DATA, $initializationVector);`

Hope this is helpful.

**Comment by damianelder** [comment]
the latest version doesn't require this.

**Comment by gaurav2262** [comment]
Yes as per the new version the encryption and decryption of the document are removed.

**Comment by zhumo1998** [comment]
> 是的，根据新版本，文件的加密和解密被删除。

Thank you very much. I have found a solution

**Comment by zhumo1998** [comment]
> 最新版本 2021-06-30 不需要这个。是的，我在旧版本上做了类似的事情。
> 
> [第1107章](https://github.com/amzn/selling-partner-api-docs/discussions/1107)

Thank you very much. I have found a solution
